<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GumboParserTestbed</class>
 <widget class="QMainWindow" name="GumboParserTestbed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,1">
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEditPHP">
      <property name="placeholderText">
       <string>PHP file</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditQS">
      <property name="placeholderText">
       <string>Query String</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLineEdit" name="lineEditPayload">
      <property name="placeholderText">
       <string>Extra form payload. Format: key1=value1&amp;key2=value2&amp;key3=value3...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QTreeView" name="treeView"/>
    </item>
    <item row="2" column="2">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGET"/>
   <addaction name="actionPOST"/>
  </widget>
  <action name="actionGET">
   <property name="text">
    <string>GET</string>
   </property>
  </action>
  <action name="actionPOST">
   <property name="text">
    <string>POST</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
